torch/CMakeLists.txt

```diff
@@ -132,6 +132,7 @@ if(USE_CUDA)
 
     list(APPEND TORCH_PYTHON_COMPILE_DEFINITIONS USE_CUDA)
     if(USE_CUDNN)
+        list(APPEND TORCH_PYTHON_LINK_LIBRARIES torch::cudnn)
```

Comment:
`TORCH_PYTHON` should not need to be aware of any CUDA/CUDNN related headers/libraries, as it should interact with the hardware via lib torch device abstractions
```suggestion
```

Reply:
Hmm, why did you mark this conversation as resolved?

Reply:
I committed  your advice before and then I marked it as resolved. But then the build failed, showing that this line is still required. Now the change has been reverted.

Reply:
The reason is that while the dependency is private, cudnn headers are exposed in public headers.